content/docs/introduction/faq.md

```diff
@@ -89,6 +89,16 @@ Yes, with the experimental [Alertmanager](https://github.com/prometheus/alertman
 Yes, with [PromDash](/docs/visualization/promdash/) and [Console
 templates](/docs/visualization/consoles/).
 
+### Can I change the timezone? Why is everything in UTC?
+
+For monitoring purposes, it is absurd to use any timezone observing
```

Comment:
Yes please. @beorn7 I know how we all feel about daylight savings time, but this is not the best place for these feelings :) I would word the whole paragraph less prescriptively (as in, "we know what's best for you"). I can understand why some people would want to look at stuff in local time in their situations, and I think "this is potentially very tedious work with little benefit" is a bit exaggerated / prescriptivist as well. I'd rather say that so far, we haven't had the need for it (and maybe explain in somewhat more neutral terms why), but we welcome UI-side contributions which would make this possible.